framework/wazuh/cdb_list.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+from os import listdir
+from os.path import isfile, isdir, join
+import re
+
+from wazuh import common
+from wazuh.exception import WazuhException
+from wazuh.utils import sort_array, search_array
+
+
+def _get_relative_path(path):
+    """
+    Get relative path
+    :param path: Original path
+    :return: Relative path (from Wazuh base directory)
+    """
+    return path.replace(common.ossec_path, '')[1:]
+
+
+def _check_path(path):
+    """
+    Check if path is well formed (without './' or '../')
+    :param path: Path to check
+    :return: Result of check the path (boolean)
+    """
+    regex_path = r'^(etc/lists/)[\w\.\-/]+$'
+    pattern = re.compile(regex_path)
```

Comment:
Compiling a regex is expensive. I would suggest defining regex at a module scope and compiling them only once.